lib/src/config_provider/config.dart

```diff
@@ -153,7 +150,8 @@ class Config {
             'Path to folder containing libclang dynamic library, used to parse C headers',
         requirement: Requirement.no,
         defaultValue: () => getDylibPath(Platform.script
-            .resolve(path.join('..', 'tool', 'wrapped_libclang'))
+            .resolve(path.posix.join('..', 'tool',
+                'wrapped_libclang')) // Path needs to be in posix style here.
```

Comment:
Either explain why it needs to be posix style (Windows issues), or remove the comment.